worlds/sc2wol/docs/en_Starcraft 2 Wings of Liberty.md

```diff
@@ -30,7 +30,7 @@ The goal is to beat the final mission: 'All In'. The config file determines whic
 ## Which of my items can be in another player's world?
 
 By default, any of StarCraft 2's items (specified above) can be in another player's world. See the
-[Advanced YAML Guide](https://archipelago.gg/tutorial/Archipelago/advanced_settings/en)
+[Advanced YAML Guide](https://archipelago.gg/tutorial/Archipelago/advanced_options/en)
```

Comment:
advanced_options doesn't exist

Reply:
Hmm good catch, I think that page needs to be changed actually

Reply:
```suggestion
[Advanced YAML Guide](https://archipelago.gg/tutorial/Archipelago/advanced_settings/en)
```